department-of-veterans-affairs/caseflow
# Add validation to Special Issues page during checkout

Resolves #13735

### Description
Ensure something is selected when user selects "Continue" button on the Special Issues page during checkout.

### Acceptance Criteria
- [x] Error banner appears when nothing is selected
- [x] Checkout still works
- [x] Checkout flow allows nothing selected when the ToggleFeature is off

### Testing Plan
1. Log in as a judge or attorney
1. Go to their queue; select an appeal
1. In the Case Details page, select "Decision ready for review" action
1. In the "Select special issues" page, click "Continue" button.
1. Note the error banner matches what is desired.
1. Select an issue or "No special issues"; click "Continue" button
1. Now you're on the next step of checkout. You can continue to make sure checkout still works or go "Back" to do more testing.

### User Facing Changes

 
### Documentation Updates
- [ ] Update documentation: training documentation to be updated
